### PR TITLE
Tlp rdw

### PR DESCRIPTION
**Pull request by adbrown101**
Added tlp-rdw as a sub-package of tlp. I hope this is ok as maintainer is listed as orphan.

### CODE REVIEW

File: srcpkgs/tlp/template [hunk 1]
```diff
@@ -35,3 +35,21 @@ do_install() {
...
+	pkg_install() {
+		make DESTDIR=${PKGDESTDIR} \
+			TLP_SBIN=/usr/bin \
+			TLP_ULIB=/usr/lib/udev \
+			TLP_SHCPL=/usr/share/bash-completion/completions \
+			TLP_ELOD=/usr/lib/elogind/system-sleep \
+			TLP_WITH_SYSTEMD=0 \
+			install-rdw
+
+	vmkdir usr/share/man/man8
```
Comment (xtraeme) [inline srcpkgs/tlp/template]: please fix indentation

Comment (adbrown101) [reply srcpkgs/tlp/template]: Indentation fixed

**Comment by Duncaen** [comment]
squash the commits, there is no point in having a `New Package:` commit for sub packages.

File: srcpkgs/tlp/template [hunk 2]
```diff
@@ -35,3 +35,21 @@ do_install() {
...
+	pkg_install() {
+		make DESTDIR=${PKGDESTDIR} \
+		    TLP_SBIN=/usr/bin \
+		    TLP_ULIB=/usr/lib/udev \
+		    TLP_SHCPL=/usr/share/bash-completion/completions \
+		    TLP_ELOD=/usr/lib/elogind/system-sleep \
+		    TLP_WITH_SYSTEMD=0 \
+		    install-rdw
+
+		vmkdir usr/share/man/man8
```
Comment (sgn) [inline srcpkgs/tlp/template]: The convention is install everything in `do_install` step.

Then move sub-package's files in `pkg_install`.

It'll be easier for inspect.

**Comment by adbrown101** [comment]
Squashed commit

File: srcpkgs/tlp/template (hunk 2 above)
Comment (adbrown101) [reply srcpkgs/tlp/template]: Yes this makes much more sense, I think I mis-understood how sub-packages are built and installed on using xbps-src. I hope the latest push is what you meant.

**Comment by adbrown101** [comment]
I will create a new pull request for the latest version

**Pull request closed without merging**